docs/getting_started/tutorial.md

```diff
@@ -119,15 +119,15 @@ When logged into the admin site, a superuser has full permissions and is able to
 python manage.py runserver
 ```
 
-If everything worked, <http://127.0.0.1:8000> will show you a welcome page:
+After running the above command,the server generates a URL <http://127.0.0.1:8000>. Copy and paste the link into your browser tab, which displays the welcome page.
 
 ![Browser screenshot of "Welcome to your new Wagtail site!" page, with teal egg above the title, and links to different resources. The page is shown inside a browswer tab, with browser URL bar at the top](../_static/images/tutorial/tutorial_1.png)
 
 ```{note}
 Throughout this tutorial we will use `http://127.0.0.1:8000` but depending on your setup, this could be `http://localhost:8000/` or a different IP address or port. Please read the console output of `manage.py runserver` to determine the correct url for your local site.
 ```
 
-You can now access the administrative area at <http://127.0.0.1:8000/admin>
+You can now access the administrative section by clicking on the **Admin Interface** on the welcome page at port <http://127.0.0.1:8000/admin>.
```

Comment:
I don't think we need this change, it's just becoming too wordy, also technically the port is the wrong term here. 

But let's just remove this change.

Reply:
Thanks for the feedback